platform/platform-app/src/main/resources/features.xml

```diff
@@ -551,6 +551,19 @@
         <bundle start-level="35">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/${cxf.jackson.version}</bundle>
         <bundle start-level="35">mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/${cxf.jackson.version}</bundle>
     </feature>
+    <feature name="cxf-jsr-json" version="${cxf.version}" install="manual">
+        <bundle start-level="35">mvn:javax.json/javax.json-api/${cxf.json.api.version}</bundle>
+        <bundle start-level="35">mvn:org.apache.johnzon/johnzon-core/${cxf.johnzon.version}</bundle>
+    </feature>
+    <feature name="cxf-tracing-brave" version="${cxf.version}" install="manual">
+        <feature version="${cxf.version}">cxf-core</feature>
+        <bundle start-level="10" dependency="true">mvn:com.google.code.findbugs/jsr305/${cxf.findbugs.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.java/zipkin/${cxf.brave.zipkin.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.reporter/zipkin-reporter/${cxf.brave.reporter.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.brave/brave/${cxf.brave.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.brave/brave-instrumentation-http/${cxf.brave.version}</bundle>
+        <bundle start-level="40">mvn:org.apache.cxf/cxf-integration-tracing-brave/${cxf.version}</bundle>
+    </feature>
```

Comment:
❓ Do we need to include these 2 new cxf features? Who starts them?

Reply:
I'm not sure who, or if anything starts these features currently. I noticed the cxf features we have in our features file are kept up to sync almost exactly with the features file in cxf, so I just added these two features to maintain keeping them our features file in sync with cxf's. I don't mind removing though if preferred.

Reply:
I would say let's not add them if we can get away with it. The less cxf features to maintain with each upgrade, the better. The code comment for all of the cxf features in this file mentions that one of the DDF changes is "removed features that are not needed", so not copying over these 2 new features could just fall under that.

Reply:
Sounds good to me, I'll remove them then.

Reply:
Removed.